site/utils/index.ts

```diff
@@ -19,3 +20,16 @@ export const redirectTo = (res: NextApiResponse | ServerResponse, location: stri
 
 export const getCsrfToken = (ctx: NextPageContext | NextPageContext): string =>
     (ctx.res as ResponseWithLocals)?.locals?.csrfToken ?? "";
+
+export const convertDateTimeToFormat = (dateOrTime: string, format: string) => dayjs(dateOrTime).format(format);
+
+export const formatTime = (time: string) => (time ? time.slice(0, -2) + ":" + time.slice(-2) : "");
+
+export const splitCamelCaseToString = (s: string) => {
```

Comment:
We're already using lodash which I believe has a function to do this for us, see https://lodash.com/docs/4.17.15#lowerCase

or https://lodash.com/docs/4.17.15#startCase 

It depends how you want to capitalisation to work.

Reply:
I want it to convert text like 'specialCase' to 'Special Case'

Reply:
@alainajaffer  thats https://lodash.com/docs/4.17.15#startCase  - but I think we likely need sentence case dont we?
like this:

'A dog went to the river' 

instead of

'A Dog Went To The River'

Reply:
looking into it I think we do need sentence case actually

Reply:
done